Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/riskdetails/RiskDetailsFragment.kt

```diff
@@ -51,8 +51,11 @@ class RiskDetailsFragment : Fragment(R.layout.fragment_risk_details) {
      * Make the links clickable and convert to hyperlink
      */
     private fun setUpWebLinks() {
-        binding.riskDetailsInformationLowriskBodyUrl
-            .convertToHyperlink(getString(R.string.risk_details_explanation_faq_body_with_link))
+        binding.riskDetailsInformationLowriskBodyUrl.setUrl(
+            R.string.risk_details_explanation_dialog_faq_body,
+            "FAQ",
```

Comment:
Why is "FAQ" not a string ressource?

Reply:
I can add it as a String source. In fact this will help us when the translated Strings to not contain the word "FAQ"

Also refer to LocationTracingStatusCardBodyTextView.kt
I used this as a reference. Perhaps the string resource can be used there as well

Reply:
If it's like @BMItter mentioned that the whole string is clickable you should use: `convertToHyperlink` instead of `setUrl`. Please provide screenshot for before and after. To check if it still looks the same as this PR should only separate text and url.

Reply:
I think I will have to revert back to the original implementation with _convertToHyperlink_  then. 

because current implementation only shows part of the String ....FAQ... as link and not the complete text as link.

@ralfgehrer  what do you think?